neurom/features/neuritefunc.py

```diff
@@ -244,6 +244,21 @@ def _seg_taper_rates(sec):
     return map_segments(_seg_taper_rates, neurites, neurite_type)
 
 
+def section_taper_rate(neurites, neurite_type=NeuriteType.all):
+    '''taper rate of the sections in a collection of neurites
+
+    Taper rate is defined here as the linear fit along a section'''
+    def _sec_taper_rate(sec):
+        '''Taper rate from fit along a section'''
+        pts = sec.points[:, COLS.XYZR]
+        diff = np.diff(pts, axis=0)
+        distance_interval = np.linalg.norm(diff[:, COLS.XYZ], axis=1)
+        distance = [0] + list(np.cumsum(distance_interval))
+        return np.polynomial.polynomial.polyfit(distance, 2 * sec.points[:, COLS.R], 1)[1]
```

Comment:
Not sure I understand the *2. If this corresponds to diameters rather than radii it should be specified and reported.